index.html

```diff
@@ -21,382 +21,453 @@
-          <h2 class="project-title">
-            Multi-Post Stories
-          </h2>
-          <div>
-            <ul class="flex-list project-date-list">
-              <li><h5>CANOPY</h5></li>
+            alt="Portfolio 4" />
+          <img class="project-img desktop right" 
+            src="assets/images/samples/desktop/Snapshoot-Portfolio4.png" 
+            alt="Portfolio 4" />
+            
+          <div class="work-description left">
+            <h2 class="project-title">
+              Uber Navigation
+            </h2>
+            <div>
+              <ul class="flex-list project-date-list">
+                <li class="mobile"><h5>CANOPY</h5></li>
+                <li class="desktop"><h5>Uber</h5></li>
+                <li>
+                  <img src="assets/images/icons/Counter.png" 
+                    alt="circle"/>
+                </li>
+                <li class="mobile"><label>Back End Dev</label></li>
+                <li class="desktop"><label>Lead Developer</label></li>
+                <li>
+                  <img src="assets/images/icons/Counter.png" 
+                    alt="circle"/>
+                </li>
+                <li class="mobile"><label>2015</label></li>
+                <li class="desktop"><label>2018</label></li>
+              </ul>
+            </div>
+            <div>
+              <p class="project-description mobile">
+                A daily selection of privately personalized reads; 
+                no accounts or sign-ups required.
+              </p>
+              <p class="project-description desktop">
+                A smart assistant to make driving more safe, efficient, and 
+                fun by unlocking your most expensive computer: your car.
+              </p>
+            </div>
+            <ul class="project-button-group">
               <li>
-                <img src="assets/images/icons/Counter.png" 
-                  alt="circle"/>
+                <span>
+                  HTML
+                </span>
+              </li>
+              <li class="desktop">
+                <span>
+                  Ruby on rails
+                </span>
               </li>
-              <li><label>End Dev</label></li>
               <li>
-                <img src="assets/images/icons/Counter.png" 
-                  alt="circle"/>
+                <span>
+                  CSS
+                </span>
+              </li>
+              <li>
+                <span>
+                  JavaScript
+                </span>
               </li>
-              <li><label>2015</label></li>
             </ul>
+            <div>
+              <button class="project-button" type="button"
+                onclick="return;">
+                See project
+              </button>
+            </div>
           </div>
+        </div>
+      </div>
+    </section>
+    <section id="about">
+      <div class="about-background flex-container">
+        <div class="about-details">
```

Comment:
- [ ] In the figma design in the About section, the left block takes `40%` width of the container, and the right block takes `60%` width but in your design both blocks take 50% of width so kindly consider fixing it because it needs to be identical to the figma design. 
your design
![w](https://github.com/chudisoft/Portfolio-Microverse/assets/96683843/fb89a141-2e33-488e-9c50-0c8c54ed8ef2)

figma design
![t](https://github.com/chudisoft/Portfolio-Microverse/assets/96683843/56e61625-49bf-49a3-8155-90b968dcc440)